<launch>
    <arg name="pluginlists_yaml" default="$(find mavros)/launch/px4_pluginlists.yaml" />
    <arg name="config_yaml" default="$(find mavros)/launch/px4_config.yaml" />
    <arg name="is_usv" value="true" />
    <arg name="is_uav" value="false" />
    <arg name="is_sim" value="false" />
    <arg name="danger_distance" value="3" />

    <include file="$(find offboard)/launch/rosbag_log.launch"/>
    <include file="$(find offboard)/launch/rviz.launch"/>
    <arg name="formation_distance" default="10" />

    <param name="danger_distance" value="$(arg danger_distance)" type="double"/>
    <node pkg = "offboard" name = "offboard" type="3usv" output="screen">
        <param name="formation_distance" value="$(arg formation_distance)" />
        <param name="is_usv" value="$(arg is_usv)" type="bool"/>
        <param name="is_uav" value="$(arg is_uav)" type="bool"/>
        <param name="is_sim" value="$(arg is_sim)" type="bool"/>
    </node>


    <node pkg = "offboard" name = "usv1" type="pclPoints" output="screen">
        <param name="is_sim" value="$(arg is_sim)" type="bool"/>
    </node>

    <include file="$(find offboard)/launch/kino_replan.launch"/>
    <node pkg = "offboard" name = "send_target" type="send_target" output="screen"></node>
    <node pkg="tf" type="static_transform_publisher" name="laser_to_map" args="0 0 0 0 0 0 laser_link map 1000"/>
</launch>
<!--roslaunch usb_cam usb_cam-test.launch -->